includes/plugins/google-site-kit/class-googlesitekit.php

```diff
@@ -85,11 +101,11 @@ private static function get_sitekit_ga4_settings_option_name() {
 	 * Get Site Kit's GA4 settings.
 	 */
 	private static function get_sitekit_ga4_settings() {
-		$option_name = self::get_sitekit_ga4_settings_option_name();
-		if ( false === $option_name ) {
+		$googlesitekit_analytics_settingsion_name = self::get_sitekit_ga4_settings_option_name();
```

Comment:
```suggestion
		$googlesitekit_analytics_settings_name = self::get_sitekit_ga4_settings_option_name();
```

Typo?

Reply:
Whoops, too broad search-replace. I did not mean to edit these lines.